pkg/actors/actors.go

```diff
@@ -1087,7 +1129,28 @@ func (a *actorsRuntime) doExecuteReminderOrTimerOnInternalActor(ctx context.Cont
 
 		err = internalAct.InvokeTimer(ctx, reminder, md)
 		if err != nil {
-			if !errors.Is(err, ErrReminderCanceled) {
+			if errors.Is(err, ErrReminderCanceled) {
```

Comment:
This code path is for timers, but we don't have timer cancellation, only reminder cancellation.